### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by Fourchaux**
Continuation of #311

### CODE REVIEW

File: src/core/CCOpt.mli [hunk 1]
```diff
@@ -23,119 +23,126 @@ val is_none : _ t -> bool
...
     @raise Invalid_argument if the option is [None]. *)
 
 val get_lazy : (unit -> 'a) -> 'a t -> 'a
-(** [get_lazy default_fn x] unwraps [x], but if [x = None] it returns [default_fn ()] instead.
+(** [get_lazy default_fn o] unwraps [o], but if [o] is [None] it returns [default_fn ()] instead.
     @since 0.6.1 *)
 
 val sequence_l : 'a t list -> 'a list t
-(** [sequence_l [x1; x2; ...; xn]] returns [Some [y1;y2;...;yn]] if
+(** [sequence_l [x1; x2; ⋯; xn]] returns [Some [y1; y2; ⋯;yn]] if
```
Comment (c-cube) [inline src/core/CCOpt.mli:99]: do you mean to use `…` instead of `···` ? :)

Comment (Fourchaux) [reply src/core/CCOpt.mli:99]: Euh, yes I mean to use … instead of . . . 
Is there a pb somewhere?

Note: maybe ⋯ would be better than ⋯; (no ; after ⋯)?

Comment (c-cube) [reply src/core/CCOpt.mli:99]: you're using · instead of . in your dots (middle of the line instead of bottom).

Comment (Fourchaux) [reply src/core/CCOpt.mli]: OK, I see.
Killing all these little beasts.

**Comment by c-cube** [comment]
thank you! :)

**Pull request merged**